src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -10,10 +10,6 @@ export default class ContentSwitcher extends React.Component {
     selectedIndex: PropTypes.number,
   };
 
-  static defaultProps = {
-    selectedIndex: 0,
```

Comment:
My concern over removing this is breaking someones code that relies upon this default behavior. Does anyone forsee this as an issue?

Reply:
Yeah, I'm a bit concerned about that as well. What's the motivation behind removing this?

Reply:
Two reasons: 
(1) The default selection is surprising, as my expectation is that if `selectedIndex` were undefined, then no item would be selected (it could be that I don't fully understand the use case for this component).

(2) To honor the use case in #409 where child components are injected into `ContentSwitcher`, where one child component is selected, and `selectedIndex` is not set. However, after thinking about this part more, a user could work around this issue by noting which of their child components is selected, and then setting the corresponding value for `selectedIndex`.

Reply:
Sorry for being late on this, but here’s my thought - `selectedIndex` is not present should mean delegating the control of selection fully in `<ContentSwitcher>`. If it’s present that should be honored (that said, I’m a bit surprised that `<ContentSwitcher>` does not have `cWRP` to update the state upon change in props). Wrt #409 I’d assume the selection state of `<Switch>` should be updated upon parent `<ContentSwitcher>`’s selection, if I’m not missing anything.

Reply:
Thanks @asudoh. Closing PR.